packages/devtools/pubspec.yaml

```diff
@@ -18,7 +18,7 @@ environment:
   sdk: '>=2.3.0 <3.0.0'
 
 dependencies:
-  devtools_server: 0.1.7
+  devtools_server: 0.1.8
```

Comment:
should these deps be 0.1.9 or 0.1.8? Since you're waiting on https://github.com/flutter/devtools/pull/1055, I would think you would want 0.1.9.

Reply:
They will be 0.1.9. Need to publish first though.

Reply:
Should be good now. PTAL.